draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2718,110 +2811,96 @@ uCDN-A                         dCDN-B                         dCDN-C
    <vspace blankLines="1"/>
    Value: A JSON string, dCDN's CDN PID, as defined in <xref target="cdn-pid"/>.
    <vspace blankLines="1"/>
-   Mandatory: Only in the collection of all triggers, if dCDN implements the filtered collections.
-         Optional in the filtered collections (uCDN can always find
-         dCDN's cdn-id in the collection of all triggers, but dCDN can choose to repeat that information
-         in its implementation of filtered collections).
+   Mandatory: No. 
    </t>
 
    </list></t>
    </list></t>
 
-      <section title="Filtered Collections by Status Link" anchor="filtered-collection-label-link">
+      <section title="Filtered Representations" anchor="filtered-collection-representation">
       <t>
-         The Filtered Collections by state Link object represents a link to a filtered collection
-        per <xref target="trigger-state">trigger state</xref>.
-         It is encoded as a JSON object containing the following name/value pairs:
+      The filtered representation object describes a particular filtered representation of triggers. 
+      Currently it supports filtering by trigger state and trigger label. In the future, further filtering capabilities may be added.
+      It is encoded as a JSON object containing the following name/value pairs:
       </t>
-           <t><list style="empty" hangIndent="3">
-      <t><list style="hanging" hangIndent="3"><t hangText="Name: status">
-           <vspace blankLines="1"/> 
-      Description: Trigger state matching the filtered collection.
-           <vspace blankLines="1"/>
-           Value: A trigger State, as defined in <xref target="trigger-state"/>.
-           <vspace blankLines="1"/>
-      Mandatory: Yes.
-           </t>
-        
-           <t hangText="Name: collection">
-           <vspace blankLines="1"/>
-      Description: Filtered collection URL corresponding to the state.
-           <vspace blankLines="1"/>
-           Value: A URL represented as a JSON string.
-           <vspace blankLines="1"/>
-      Mandatory: Yes.
-           </t>
-        
-      </list></t>
-           </list></t> 
-
-      </section>
+   
 
-      <section title="Filtered Collections by Label Link" anchor="filtered-collection-state-link">
-      <t>
-         The Filtered Collections by Label Link object represents a link to a filtered collection
-              per trigger label.
-         It is encoded as a JSON object containing the following name/value pairs:
+      <t><list style="empty" hangIndent="3">
+      <t><list style="hanging" hangIndent="3">
+      <t hangText="Name: state">
+      <vspace blankLines="1"/> 
+      Description: Trigger state matching the filtered collection representation.
+      <vspace blankLines="1"/>
+      Value: A trigger state, as defined in <xref target="trigger-state"/>.
```

Comment:
May need some capitalization fixes here.  I'm seeing "Trigger state" and "trigger state", which does not match "Trigger Label" below.

Reply:
Agree
This is the original draft style, using capitalization, I am not a fan of it.
Will review across the board and revise.

Reply:
Done